### DISCUSSION TITLE
Welcome to IGFI Discussions!

### DISCUSSION THREAD

**Discussion by new92** [new92/igfi#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mgiliberto** [comment]
Getting an error: LoginRequired: login_required

**Reply by new92** [reply]
Okay so in order to resolve that bug you have to login to your Instagram account in which you want to increase the followers.

**Reply by mgiliberto** [reply]
Thanks for the reply.  Yes I chose 1 for Increase Follwers and when it went through the follow process I recieved that error after I followed everyone.

**Reply by mgiliberto** [reply]
![Screenshot 2023-03-21 164914](https://user-images.githubusercontent.com/69218015/226737428-87ebf8f7-5336-4edf-89a3-19d7e6cfa5f7.png)

**Reply by new92** [reply]
You are welcome ! I am glad I helped ! If you have any other questions/suggestions feel free to reach out to me at anytime !

**Reply by mgiliberto** [reply]
Sorry but I meant I am still  getting the error.  The login works and I start to follow the list but then it stops.  See screenshot.

**Reply by new92** [reply]
Oh okay sorry my bad. I misunderstood, okay so try to clone again the repository I've made some changes, and let me know

**Reply by mgiliberto** [reply]
Same error however your v2 did work...

**Reply by new92** [reply]
Ok please use temporarily V2, while I'm reparing V3

**Reply by mgiliberto** [reply]
thank you

**Reply by mgiliberto** [reply]
also: is this correct?

[✓] Successfully followed: 19 users
[✓] Successfully unfollowed: 0 users
[✕] Failed to unfollow: 19 users
[+] Percentage of success: 0.2638888888888889
[+] Percentage of fail: 99.73611111111111%
[✓] Successfully added: 0 followers.

**Reply by new92** [reply]
No it's not correct. So there is the file [IMPORTANT.md](https://github.com/new92/IGFollowersIncreaser/blob/main/IMPORTANT.md) which answers your question. But the answer is very simple: Instagram detects when the followers are being increased using the technique (which the scripts are using). So when they detect too much activity they block the account. So in order for your account not to get blocked you use the script every 2-3 days.
